gofannon/open_notify_space/iss_locator.py

```diff
@@ -0,0 +1,126 @@
+import jsonschema
+import jsonschema.exceptions
+from ..base import BaseTool
+from ..config import FunctionRegistry
+
+import logging
+import json
+
+import requests
+from jsonschema import validate
```

Comment:
@jolson-ibm awesome add here
@rawkintrevo make issue to expand this to other tools